node/src/Pos/Communication/Message.hs

```diff
@@ -41,10 +41,13 @@ instance Message (MsgBlock ssc) where
     messageCode _ = 7
     formatMessage _ = "Block"
 
-instance Message MsgSubscribe where
+instance Message MsgSubscribe1 where
     messageCode _ = 13
-    formatMessage _ = "Subscribe"
+    formatMessage _ = "Subscribe1"
 
+instance Message MsgSubscribe where
+    messageCode _ = 14
+    formatMessage _ = "Subscribe"
```

Comment:
I find it strange that `MsgSubscribe1` is the *old* message type. Shouldn't it be the new one? If we make another change, the newer one will be `MsgSubscribe`, the second-oldest will be `MsgSubscribe2`, and the oldest `MsgSubscribe1`? Anyway, just a matter of taste...

Reply:
> If we make another change, the newer one will be MsgSubscribe, the second-oldest will be MsgSubscribe2, and the oldest MsgSubscribe1?

Yes. I thought it would be the clearest option - whenever you see a type with no numbers, it's the newest one. That's a matter of taste/convention however.

Reply:
> whenever you see a type with no numbers, it's the newest one

I'm happy with that convention. It's the same one we use in the acid-state versioning stuff.

Reply:
@coot If there is another version of MsgSubscribe, then you'd need to add listener for the newest type version, so you can't really forget to do that.